homeassistant/components/xbox/strings.json

```
@@ -118,6 +124,9 @@
     "account_not_configured": {
       "message": "The Xbox account is not configured."
     },
+    "auth_exception": {
+      "message": "Xbox Network authentication failed, re-authentication required"
```

```suggestion
      "message": "Xbox account authentication failed, re-authentication required"
```

I don't believe we have the "network" elsewhere